theme/static/js/hs_file_browser.js

```diff
@@ -690,14 +690,15 @@ function bindFileBrowserItemEvents() {
             var fileExtension = fileName.substr(fileName.lastIndexOf("."), fileName.length);
 
             // toggle apps by file extension and aggregations
+            let hasTools = false;
             menu.find("li.btn-open-with").each(function() {
                 var agg_app = false;
-                if ($(this).attr("agg-types")){
-                    agg_app = $.inArray(fileAggType, $(this).attr("agg-types").split(",")) !== -1;
+                if ($(this).attr("data-agg-types")){
+                    agg_app = $.inArray(fileAggType, $(this).attr("data-agg-types").split(",")) !== -1;
                 }
```

Comment:
are there ever whitespaces surrounding the comma warranting trim() or do you control those strings?

Reply:
can you use something like includes() or would that break it (partial matches)?